bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -376,11 +382,15 @@ async fn extension_loop_active(
         .to_string();
     let tags_provider = setup_tag_provider(&Arc::clone(&aws_config), config, &account_id);
 
+    let (logs_aggr_service, logs_aggr_handle) = LogsAggregatorService::default();
+    start_logs_aggregator(logs_aggr_service);
+
```

Comment:
I think this logic can live in the `start_logs_agent` as no one else uses this but logs related stuff